package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Thu Feb 15 16:16:19 UTC 2018 - ancor@suse.com
+
+- Adjusted the suggested and minimum sizes of all the booting
+  partitions, both in the storage proposal and in the Partitioner
+  validations (bsc#1076851 and fate#318196).
```

Comment:
NP: comment something about weight for booting partitions?

Reply:
Weight is a rather low-level detail. The important user-facing effect is that the suggested sizes have changed. And that is mentioned.